department-of-veterans-affairs/va.gov-cms
# VACMS-4923: Add service limiting logic to service dropdowns.

## Description
See #4923 (this is a redo, as first round didn't account for edit forms as much as thy should have)

## Testing done
Visual / behat

## QA steps
- As an administrator, go to `user/34/workbench_access` and assign a second section to user.
- Login as user 34, and go to `node/add/health_care_local_health_service`
- Change Owner field to Pittsburgh, and then visually verify that system and facility service selects only show Pittsburgh services.
- Change Owner field to other section, and then visually verify that system and facility service selects only show second section.
- Change Owner field to `- Select a value -` and visually verify that system and facility service selects show `- Select a value -`
- As an administrator, visit an existing Facility health service node, e.g.: `node/19634/edit` and visually verify that the dropdown default values match the facility.

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
